Chat/WebApi/Program.cs

```diff
@@ -0,0 +1,84 @@
+using Application.Dtos;
+using Application.Extentions;
+using Application.Ports.Services;
+using Application.Services;
+using Application.Validators;
+using Domain.Interfaces;
+using FluentValidation;
+using Infrastructure.Data;
+using Infrastructure.Repositories;
+using Microsoft.EntityFrameworkCore;
+using FluentValidation.AspNetCore;
+using Application.AutoMapperProfiles;
+
+namespace WebApi
+{
+    public class Program
+    {
+        public static void Main(string[] args)
```

Comment:
Clean up this class by extracting everything into extension methods and then linking them